spec/shipcloud/address_spec.rb

```diff
@@ -35,23 +35,27 @@
 
   describe '.create' do
     it 'makes a new POST request using the correct API endpoint' do
-      Shipcloud.should_receive(:request).with(:post, 'addresses', valid_attributes).and_return('data' => {})
+      expect(Shipcloud).to receive(:request).
+        with(:post, 'addresses', valid_attributes, api_key: nil).and_return('data' => {})
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.